src/app/models/proptypes.js

```diff
@@ -1,92 +0,0 @@
-import { arrayOf, bool, oneOfType, shape, string } from 'prop-types';
```

Comment:
This has been moved to the dir of the same name as the index.js

This dir is very messy. This PR will not refactor it.